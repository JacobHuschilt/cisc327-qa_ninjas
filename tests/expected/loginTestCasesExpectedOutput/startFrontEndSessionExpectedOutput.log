Error! Missing arguments for command: java.lang.ArrayIndexOutOfBoundsException: 1