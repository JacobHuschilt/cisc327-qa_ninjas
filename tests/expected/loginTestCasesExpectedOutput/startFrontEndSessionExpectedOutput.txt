Error! Not logged in
Error: Already Logged-out!